tests/windows_test.go

```diff
@@ -159,21 +156,23 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
-	Context("[ref_id:139]VMI is created", func() {
+	JustBeforeEach(func() {
+		By("Starting the windows VirtualMachineInstance")
+		var err error
+		windowsVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(windowsVMI)
+		Expect(err).To(BeNil())
+		tests.WaitForSuccessfulVMIStartWithTimeout(windowsVMI, 360)
```

Comment:
Just keep in mind that `JustBeforeEach` needs to be used with care. Using it wrongly could lead to problems that could be very hard to debug. Personally I'm not sure that using this worth the risk. AFAICT the only benefit is avoiding duplicated code, which we can also do by introducing a function.

To this specific case - you're using 360 seconds timeout here, while the `BeforeEach` under `Context("with bridge binding"...` defines 420 seconds.

I would avoid introducing a function instead of `JustBeforeEach` and call it from the different `BeforeEach`s. Something like `createAndStartVMI(timeout)`. WDYT?

Reply:
I will remove the commit with this change, because it would be changed anyway in my future PR.